core/mutations/UpdateArticle.js

```diff
@@ -0,0 +1,9 @@
+import gql from "graphql-tag";
+
+export default gql`
+  mutation updateArticle($id: ID, $title: String, $body: String) {
+    updatedArticle(articleInput: { id: $id, title: $title, body: $body }) {
```

Comment:
Rename operation to be imperative like `updateArticle`

Reply:
Also, see the  comment about using Input as arguments before.

Reply:
Here it changed to updatedArticle since it returns the article, I tried to follow the same idea of the field accessed value for the queries, do I revert this?

Reply:
Yes please ! 
More info: https://blog.apollographql.com/designing-graphql-mutations-e09de826ed97